idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/intention/IntentionSyntax.kt

```diff
@@ -142,23 +138,6 @@ interface IntentionSyntax : IntentionUtilitySyntax {
       override fun isApplicableForCodeFragment(): Boolean =
         isApplicableForCodeFragment
     }
-
-  /* TODO: Deprecated until MetaData is in the Functionparameters
-  /**
-   * [addIntentionWithMetaData] extension registers an Intention with Metadata.
-   * There is an important note to this function.
-   * The internals of IntelliJ-based Ides will bail with an Exception, if the required `MetaData` - Html and intentionDescription - is not in the right folder within `resources`.
-   * The `MetaData` defines a Html file with a description of the Intention and under the path `/resource/intentionDescription/<IntentionName>` one `after` and `before` template file.
-   * Bare with us we will improve upon this issue at a later point.
-   * @param category are used to order Intentions
-   * @see com.intellij.codeInsight.intention.IntentionManager.registerIntentionAndMetaData
-   * @sample org.jetbrains.kotlin.idea.intentions.ValToObjectIntention
-   */
-  fun IdeMetaPlugin.addIntentionWithMetaData(
-    category: String,
-    intention: IntentionAction
-  ): ExtensionPhase =
-    IntentionExtensionProvider.RegisterIntentionWithMetaData(intention, category)*/
```

Comment:
Should class `RegisterIntentionWithMetaData` be kept? It's still used in `IdeInternalRegistry`.

Reply:
Thanks :)